packages/compartment-mapper/src/import.js

```diff
@@ -1,7 +1,7 @@
 // @ts-check
 /* eslint no-shadow: "off" */
 
-/** @import {Application} from './types.js' */
+/** @import {Application, ExtraImportOptions} from './types.js' */
```

Comment:
The “extra” qualifier here isn’t doing a bunch of work.
```suggestion
/** @import {Application, ImportOptions} from './types.js' */
```

Reply:
I will rename this.